unstoppable-ios-app/domains-manager-ios/Extensions/UIImage.swift

```diff
@@ -121,6 +121,7 @@ extension UIImage {
     static let arrowRight = UIImage(named: "arrowRight")!
     static let timeIcon24 = UIImage(named: "timeIcon24")!
     static let dragIcon24 = UIImage(named: "dragIcon24")!
+    static let parkingIcon24 = UIImage(named: "parkingIcon24")!
```

Comment:
Maybe extract these icons?

Reply:
This is place where all extracted icons placed